doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,2 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
@Ricknox Sorry, I meant if singular `src/Migration` as the default directory would be more appropriate here (as the migrations are php classes, and most of the libraries use singular names in their namespace to describe the domain, fe https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.6/etc/packages/doctrine.yaml#L12)